lib/times_table.js

```diff
@@ -0,0 +1,20 @@
+
+function timesTable(num) {
+  if (num <= 0) {
+    return null;
+  }
+
+  var display = ""
+
+  for (var i = 1; i <= num; i++) {
+    for (var j = 1; j <= num; j++) {
+      var multiple = i * j;
+      display += multiple + " ";
+    }
```

Comment:
To avoid nested for loops like this, try leveraging some Array.prototype methods